_quarto.yml

```
@@ -19,7 +19,11 @@ website:
         href: directory.qmd
         aria-label: "Globe icon: Link to directory page of other SDCA websites"
 
-format: sdca-theme-html
+format:
+  html:
```

```suggestion
  sdca-theme-html:
```

This is what I meant. The format has to match what the extension is providing. So it has to be the same name as the extension.

---

Yes, it does work. Because `html` is a format that exists. But it means we don't have access to using `emojis` as an example (see line 9 of the extension).